pom.xml

```diff
@@ -90,8 +96,29 @@
 		<dependency>
 			<groupId>org.jenkins-ci.modules</groupId>
 			<artifactId>sshd</artifactId>
-			<version>1.6</version>
+			<version>2.2</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.jenkins-ci.plugins</groupId>
+			<artifactId>script-security</artifactId>
+			<version>1.34</version>
+		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-all</artifactId>
```

Comment:
Generally it's recommended not to depend on mockito-all. Maybe mockito-core is enough here.
FTR https://solidsoft.wordpress.com/2012/09/11/beyond-the-mockito-refcard-part-3-mockito-core-vs-mockito-all-in-mavengradle-based-projects/